src/interpreter/plugin/FunctionPlugin.ts

```diff
@@ -355,7 +355,7 @@ export abstract class FunctionPlugin implements FunctionPluginTypecheck<Function
     metadata: FunctionMetadata,
     fn: (...arg: any) => InterpreterValue,
   ) => {
-    let argumentDefinitions: FunctionArgument[] = metadata.parameters!
+    let argumentDefinitions: FunctionArgument[] = metadata.parameters || []
```

Comment:
This is a bugfix. Our documentation says that `parameters` field is optional, but this function fails when it's not defined.